plane/source/docs/qacro-mode.rst

```diff
@@ -0,0 +1,30 @@
+.. _qacro-mode:
+
+=========+
+QACRO Mode
+=========+
+
+QACRO (for acrobatic) is a quadplane mode for advanced users that provides rate
+based stabilization like :ref:`Copter ACRO <copter:acro-mode>`.  QACRO mode is best suited for aerobatic flight of a copter tailsitter.
+
+It is recommended to set parameter Q_TAILSIT_INPUT to FixedWing (1) for this mode. This selects plane style controls with the aileron stick controlling body frame roll rate and the rudder stick controlling body frame yaw rate.
+
+Also, set Q_M_HOVER_LEARN to 2 (learn and save) to allow the throttle midpoint to set automatically.
+
+Training mode is not implemented and this mode will behave similarly to a fixed wing plane in MANUAL mode, with no limitations on earth frame roll, pitch and yaw.
+
+Aileron and elevator stick scalings are set by parameters ACRO_ROLL_RATE and ACRO_PITCH_RATE, respectively, with default of 180 deg/sec. Yaw stick scaling is set by Q_YAW_RATE_MAX with default of 100 deg/sec.
+
+To avoid oscillation, it is necessary to reduce control surface deflections at high airspeeds in VTOL modes.  If the vehicle has no airspeed sensor, this reduction is based on attitude (tilt angle from vertical) and throttle setting, since these are generally correlated with airspeed. If an airspeed sensor is available, the reduction is based on the measured airspeed.
```

Comment:
i think this needs a sub title, "For Tailsitters" or something

Reply:
I'm working on a PR to remove the confusion between roll and yaw, and add a new parameters as necessary to get the units right.